src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Formatting/RazorFormattingPass.cs

```diff
@@ -398,7 +398,8 @@ static int GetAdditionalIndentationLevel(FormattingContext context, Range range,
                 if (!context.TryGetIndentationLevel(codeNode.Position, out var desiredIndentationLevel))
                 {
                     // If for some reason we don't match a particular span use the indentation for the whole line
-                    var indentation = context.Indentations[range.Start.Line];
+                    var indentations = context.GetIndentations();
+                    var indentation = indentations[range.Start.Line];
```

Comment:
Should we log in this case? Seems like if we're getting into "If for some reason" we should identify the scenario for further research.

Reply:
What case exactly? I just extracted the indentations to a local, none of the actual code changed.